File: android-kiosco/app/src/main/java/com/hoteldelvalle/kiosco/Plan.java
package com.hoteldelvalle.kiosco;

public class Plan {
    private final String key;
    private final String name;
    private final String subtitle;
    private final String icon;
    private final int price;
    private final boolean hero;
    private final String badge;

    public Plan(String key, String name, String subtitle, String icon, int price, boolean hero, String badge) {
        this.key = key;
        this.name = name;
        this.subtitle = subtitle;
        this.icon = icon;
        this.price = price;
        this.hero = hero;
        this.badge = badge;
    }

    public String getKey() { return key; }
    public String getName() { return name; }
    public String getSubtitle() { return subtitle; }
    public String getIcon() { return icon; }
    public int getPrice() { return price; }
    public boolean isHero() { return hero; }
    public String getBadge() { return badge; }
}

File: android-kiosco/app/src/main/java/com/hoteldelvalle/kiosco/data/model/Plan.kt
// Dep: modelo estático de planes (sin serialización)
package com.hoteldelvalle.kiosco.data.model

data class Plan(
    val key: String,
    val name: String,
    val subtitle: String,
    val icon: String,
    val price: Int,
    val isHero: Boolean,
    val badge: String?
) {
    companion object {
        val PLANS = listOf(
            Plan("momento", "Momento", "3 horas", "M", 10, true, "El más pedido"),
            Plan("amanecida", "Amanecida", "18:00 a 9:00", "A", 20, false, null),
            Plan("hospedaje", "Hospedaje", "Múltiples días", "H", 30, false, null),
            Plan("suite", "Suite Jacuzzi", "3 horas · Jacuzzi privado", "S", 20, false, null),
        )
    }
}
